Marsiple/Common/Network/MartianApi.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MatrianApi.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+struct MartianApi {
+    static let url = "https://demo.martian.agency/api"
+    
+    struct Headers {
```

Comment:
Hmm, I don't think mixing `HTTPHeaders` with possible values is a good idea. Please isolate them into separate structs. 

In example, `HTTPHeaders` could be modelled with a struct `HTTPRequestHeaders`, containing header keys only.